### PR TITLE
n6010 image info

### PR DESCRIPTION
**Pull request by hundeboll-silicom**
This depends on https://github.com/OPAE/opae-libs/pull/233

### CODE REVIEW

**Review by Anders5-dk — DISMISSED** [review]
I think you mean "Prints fpga image info" insrtead of "Prints fpga info info".

**Comment by hundeboll-silicom** [comment]
> I think you mean "Prints fpga image info" insrtead of "Prints fpga info info".

I updated the commit to fix the typo, and also updated the wrong return value description.

File: tools/fpgainfo/board.c [hunk 1]
```diff
@@ -690,3 +690,25 @@ fpga_result fpga_boot_info(fpga_token token)
...
+fpga_result fpga_image_info(fpga_token token)
+{
+	fpga_result res = FPGA_OK;
+	void *dl_handle = NULL;
+
+	fpga_result (*fpga_image_info)(fpga_token token);
+
+	res = load_board_plugin(token, &dl_handle);
+	if (res != FPGA_OK) {
+		OPAE_MSG("Failed to load board plugin\n", dlerror());
```
Comment (tswhison) [inline tools/fpgainfo/board.c:703]: This needs a format specifier (%s) for the dlerror().  I think that dlerror() can also return NULL, so there needs to also be a NULL check.

Comment (hundeboll-silicom) [reply tools/fpgainfo/board.c:703]: I've updated the format string, and added a NULL check

Comment (tswhison) [reply tools/fpgainfo/board.c:703]: Could you push the changes?  I don't see them.

Comment (hundeboll-silicom) [reply tools/fpgainfo/board.c]: Sorry, the change was still chilling in my working tree :(

**Review by tswhison — APPROVED** [review]

**Pull request merged**